assembler/templates/search.html

```diff
@@ -1,57 +1,55 @@
 <!-- /wp:spacer -->
 
-<!-- wp:query {"queryId":0,"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true,"taxQuery":null,"parents":[]},"tagName":"main","layout":{"type":"constrained","wideSize":"1200px"}} -->
-<main class="wp-block-query"><!-- wp:query-title {"type":"search","level":2,"fontSize":"large"} /-->
+<!-- wp:query-title {"type":"search","textAlign":"center","level":2,"showSearchTerm":false,"fontSize":"large"} /-->
+
+<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
+<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
 
 <!-- wp:search {"showLabel":false,"buttonPosition":"button-inside","buttonUseIcon":true,"style":{"border":{"top":{"color":"#dfdfdf","style":"solid","width":"1px"},"right":{"color":"#dfdfdf","style":"solid","width":"1px"},"bottom":{"color":"#dfdfdf","style":"solid","width":"1px"},"left":{"color":"#dfdfdf","style":"solid","width":"1px"}}},"backgroundColor":"primary","textColor":"background","fontSize":"small"} /-->
 
-<!-- wp:group {"style":{"spacing":{"padding":{"top":"20px"}}},"layout":{"type":"default"}} -->
-<div class="wp-block-group" style="padding-top:20px"><!-- wp:post-template {"layout":{"type":"grid","columnCount":3}} -->
-<!-- wp:group -->
-<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
-<div class="wp-block-group"><!-- wp:group {"style":{"border":{"radius":"4px","top":{"style":"none","width":"0px"},"right":{"style":"none","width":"0px"},"bottom":{"style":"none","width":"0px"},"left":{"style":"none","width":"0px"}}},"backgroundColor":"tertiary","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"center"}} -->
-<div class="wp-block-group has-tertiary-background-color has-background" style="border-radius:4px;border-top-style:none;border-top-width:0px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-style:none;border-left-width:0px"><!-- wp:post-terms {"term":"category","textAlign":"center","separator":"  ","style":{"layout":{"selfStretch":"fill"},"spacing":{"padding":{"top":"6px","right":"6px","bottom":"6px","left":"6px"}}}} /--></div>
-<!-- /wp:group --></div>
-<!-- /wp:group -->
+<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
+<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
 
-<!-- wp:post-featured-image {"isLink":true,"aspectRatio":"3/2"} /-->
+<!-- wp:query {"queryId":0,"query":{"perPage":10,"pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-query alignwide"><!-- wp:query-no-results -->
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">No search results.</p>
```

Comment:
We should make this into a pattern to reuse across templates, and wrap this text in translation function.